.circleci/integration-tests/Dockerfile

```diff
@@ -71,6 +112,19 @@ RUN curl -o kubectl https://s3.us-west-2.amazonaws.com/amazon-eks/1.22.6/2022-03
     && chmod +x ./kubectl \
     && mv ./kubectl /usr/local/bin
 
+#install Apache Hive (Download same version of library to match EMR hive version created by DAG)
+RUN curl "https://downloads.apache.org/hive/$HIVE_LIBRARY_VERSION/apache-$HIVE_LIBRARY_VERSION-bin.tar.gz" -o  "/tmp/apache-$HIVE_LIBRARY_VERSION-bin.tar.gz"  \
+    && tar -xf /tmp/apache-$HIVE_LIBRARY_VERSION-bin.tar.gz -C /usr/local
+ENV HIVE_HOME=/usr/local/apache-$HIVE_LIBRARY_VERSION-bin
+
+#install Apache Hadoop (Download same version of library to match EMR hadoop version created by DAG)
+RUN curl "https://archive.apache.org/dist/hadoop/common/$HADOOP_LIBRARY_VERSION/$HADOOP_LIBRARY_VERSION.tar.gz" -o "/tmp/$HADOOP_LIBRARY_VERSION.tar.gz" \
+    && tar -xf /tmp/$HADOOP_LIBRARY_VERSION.tar.gz -C /usr/local
+ENV HADOOP_HOME=/usr/local/$HADOOP_LIBRARY_VERSION
+ENV HADOOP_CONF_DIR=$HADOOP_HOME/conf
+
+ENV PATH $PATH:$JAVA_HOME/bin:$HIVE_HOME:$HIVE_HOME/bin:$HADOOP_HOME:$HADOOP_HOME/bin
```

Comment:
Do we also need `$HIVE_HOME` and `$HADOOP_HOME` in the path. Mean to ask that isn't the paths for binaries (`$HIVE_HOME/bin` &  `HADOOP_HOME/bin`) enough?